Islandora-CLAW/islandora_demo
# Solr Search Block

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/953

# What does this Pull Request do?

Adds a solr search block to the second sidebar.

# What's new?
Exported config for the solr search view page and its exposed filters as a block.

# How should this be tested?

- Spin up a new vagrant
- You should not have a search block
- Pull in this PR
- `drush fim -y islandora_demo`
- `drush cr`
- You should have a search block
![screenshot from 2019-01-17 14-53-20](https://user-images.githubusercontent.com/20773151/51341445-b577ed00-1a67-11e9-9405-e62bafc483dd.png)

# Interested parties
@Natkeeran  @Islandora-CLAW/committers
